src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -783,6 +783,11 @@ internal bool IsAutoProperty
                 return null;
             }
         }
+
+        internal bool GetMethodIsEquivalentToBackingFieldRead => _getMethod?.BodyShouldBeSynthesized == true;
```

Comment:
>GetMethodIsEquivalentToBackingFieldRead [](http://example.com/codeflow?start=22&length=39)

Why not make it an API on an accessor? Next to ```BodyShouldBeSynthesized```.

Reply:
@AlekseyTs How would this be different than `BodyShouldBeSynthesized`? If it's not suitable in `SourcePropertySymbolBase`, then we can just delete it and call `BodyShouldBeSynthesized` directly? Anything I'm missing?

Reply:
> Anything I'm missing?

I think it would be better to have a specific abstraction in place even if right now it is going to simply delegate to the other helper. Theoretically, the implementation could be more complicated. For example, we could decide to actually examine what user-defined implementation does.